### PR TITLE
Check before prod

### PR DESCRIPTION
**Pull request by ogroppo**

### CODE REVIEW

File: src/components/Node/Node.js [hunk 1]
```diff
@@ -103,7 +103,7 @@ export default function Node({
               onClick={() => setShowModal(true)}
               title={node.data.label ? `Show ${node.data.label} details` : null}
             >
-              {node.data.birthName && toggleIcons ? node.data.birthName :  node.data.label ? node.data.label : <i>Unlabelled</i>  }
+              {node.data.birthName && showBirthName ? node.data.birthName :  node.data.label ? node.data.label : <i>Unlabelled</i>  }
```
Comment (ogroppo) [inline src/components/Node/Node.js:106]: 🥳

**Comment by ogroppo** [comment]
@mshd have a look in 5 if the image server is reachable from prod

**Comment by mshd** [comment]
Yes thanks it's working see https://www.entitree.com/en/family_tree/Sudono_Salim

**Comment by ogroppo** [comment]
This is massive improvement, well done!

**Pull request merged**